aws/resource_aws_emr_instance_group_test.go

```diff
@@ -231,11 +267,49 @@ resource "aws_main_route_table_association" "a" {
   route_table_id = "${aws_route_table.r.id}"
 }
 
+## EMR Cluster Configuration
+resource "aws_emr_cluster" "tf-test-cluster" {
+  name          = "tf-test-emr-%[1]d"
+  release_label = "emr-4.6.0"
+  applications  = ["Spark"]
+
+  ec2_attributes {
+    subnet_id                         = "${aws_subnet.main.id}"
+    emr_managed_master_security_group = "${aws_security_group.allow_all.id}"
+    emr_managed_slave_security_group  = "${aws_security_group.allow_all.id}"
+    instance_profile                  = "${aws_iam_instance_profile.emr_profile.arn}"
+  }
+
+  master_instance_type = "m4.large"
+  core_instance_type = "m4.large"
+  core_instance_count = 2
+
+  tags = {
```

Comment:
As a general practice, I would recommend removing any optional arguments that are not pertinent to the acceptance test to reduce the test configuration size.